### ISSUE TITLE
ConnectionError(ProtocolError('Connection aborted.', OSError(0, 'Error')),)

### ISSUE THREAD

**Issue by ghost** [s4w3d0ff/python-poloniex#163]
Hello,

My script runing now from a long time has been stop

and i have had this error:

ConnectionError(ProtocolError('Connection aborted.', OSError(0, 'Error')),)

What is mean ? 

Thanks.

**Comment by s4w3d0ff** [comment]
Do you have a timeout set? if not maybe set it to something big, like 300.

**Comment by yepla** [comment]
yes s4w3d0ff i have a timeout set to 3.05 to connection and None for answer

**Comment by s4w3d0ff** [comment]
I'm not sure where that error is coming from, can you post the traceback?

**Comment by yepla** [comment]
i don't have it anymore but if it s happen again yes
the only problem is that i start my script in a screen so i won't be all the traceback :(

**Comment by s4w3d0ff** [comment]
Looking back, if this issue keeps happening (and you think it is this module that is causing it) you can log the `logging` output to a .txt file that you can review later when an error occurs. There are ways to truncate the file so you don't have a huge file to look through.

https://stackoverflow.com/a/40088591/3389859
```python
import logging
import poloniex
from logging.handlers import RotatingFileHandler

logging.basicConfig(filename="my_log.log", level=logging.DEBUG)
poloniex.logger.setLevel(logging.DEBUG)
fileLoggerHandler = RotatingFileHandler('my_log.log', maxBytes=1000, backupCount=2)
poloniex.logger.addHandler(fileLoggerHandler)
```

While looking for a more simple solution I found this SO question:
https://stackoverflow.com/questions/1508467/log-exception-with-traceback

One answer might work for you if you are executing your script remote using bash and want to log just the exceptions:
https://stackoverflow.com/a/36633733/3389859

Append errors to file, other output to the terminal:
```
./test.py 2>> my_log.log
```
